client/app/data/dashboard/industry_user.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "title": "My Operator",
+    "content": "View or update your operator information.",
```

Comment:
Non-admin industry users will only have view privileges for the My Operator page